asf_floorplan_interpreter/pipeline/train_yolo.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+try:
+    os.system("apt-get update && apt-get install -y libgl1-mesa-glx 1> /dev/null")
```

Comment:
I always get an error that this doesn't work. It's not an issue because of the 'try' statement, but is it worth a line of explanation for what this does/ who would need it to work?

Reply:
yes - good point! It will always error, but its needed for the batch step. Will add a comment